php_fpm/tox.ini

```diff
@@ -1,12 +1,19 @@
 [tox]
 minversion = 2.0
+skip_missing_interpreters = true
 basepython = py27
-envlist = unit, integration, flake8
+envlist =
+    unit,
+    {py27,py36}-integration,
+    flake8
 
 [testenv]
 platform = linux|darwin|win32
+passenv =
+    DOCKER*
+    COMPOSE*
 deps =
-    ../datadog_checks_base
+    -e../datadog_checks_base
```

Comment:
Argh, I'm afraid I'm missing a bunch of `-e` in several places 🤦‍♂️ 
Good catch!